vidarrtest-regression.json.in

```diff
@@ -81,7 +84,7 @@
             {
                 "metrics_calculate": "@CHECKOUT@/tests/calculate.sh",
                 "metrics_compare": "@CHECKOUT@/tests/compare.sh",
-                "output_metrics": "/.mounts/labs/gsi/testdata/guppy/output_metrics/guppyTest.metrics",
```

Comment:
there is no sensitive information here, to keep it in the same theme with the other tests I recommend reverting to the original name